package.json

```diff
@@ -73,6 +73,7 @@
     "nyc": "15.1.0",
     "performance-now": "2.1.0",
     "publish-please": "5.5.2",
+    "queue-microtask": "^1.2.2",
```

Comment:
what's up with this ?

Reply:
It's required by `@typescript-eslint/parser`, but I can fix this.

![image](https://user-images.githubusercontent.com/35312043/110114113-66f74b00-7db4-11eb-86de-046e6a15f4b0.png)